src/main/java/org/zone/region/flag/player/entitydamage/EntityDamagePlayerFlag.java

```diff
@@ -0,0 +1,54 @@
+package org.zone.region.flag.player.entitydamage;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.zone.region.flag.Flag;
+import org.zone.region.flag.FlagType;
+import org.zone.region.flag.FlagTypes;
+import org.zone.region.group.key.GroupKey;
+import org.zone.region.group.key.GroupKeys;
+
+import java.util.Optional;
+
+public class EntityDamagePlayerFlag implements Flag.TaggedFlag, Flag.AffectsPlayer, Flag.Enabled {
+
+    public @Nullable Boolean enabled;
+    public static final EntityDamagePlayerFlag ELSE = new EntityDamagePlayerFlag(false);
```

Comment:
No need for a static default when it comes to a tag flag

Reply:
kay